v0/destinations/ga4/utils.js

```diff
@@ -237,11 +237,11 @@ const GA4_ITEM_EXCLUSION = [
 ];
 
 /**
- * get items properties for a event.
+ * get product list properties for a event.
```

Comment:
Make the inline docs bit more expressive

Reply:
Done